tests/acceptance/features/apiSpaces/shareOperations.feature

```diff
@@ -0,0 +1,241 @@
+@api
+Feature: sharing
+
+  Background:
+    Given using OCS API version "1"
+    And these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+
+
+  Scenario: Correct webdav share-permissions for received file with edit and reshare permissions
+    Given using spaces DAV path
+    And user "Alice" has uploaded file with content "foo" to "/tmp.txt"
+    And user "Alice" has shared file "/tmp.txt" with user "Brian"
+    And user "Brian" has accepted share "/tmp.txt" offered by user "Alice"
+    When user "Brian" gets the following properties of file "/tmp.txt" inside space "Shares Jail" using the WebDAV API
+      | propertyName          |
+      | ocs:share-permissions |
+    Then the HTTP status code should be "207"
+    And the single response should contain a property "ocs:share-permissions" with value "19"
+
+
+  Scenario: Correct webdav share-permissions for received group shared file with edit and reshare permissions
+    Given using spaces DAV path
+    And group "grp1" has been created
+    And user "Brian" has been added to group "grp1"
+    And user "Alice" has uploaded file with content "foo" to "/tmp.txt"
+    And user "Alice" has created a share with settings
+      | path        | /tmp.txt          |
+      | shareType   | group             |
+      | permissions | share,update,read |
+      | shareWith   | grp1              |
+    And user "Brian" has accepted share "/tmp.txt" offered by user "Alice"
+    When user "Brian" gets the following properties of file "/tmp.txt" inside space "Shares Jail" using the WebDAV API
+      | propertyName          |
+      | ocs:share-permissions |
+    Then the HTTP status code should be "207"
+    And the single response should contain a property "ocs:share-permissions" with value "19"
```

Comment:
```suggestion
    And the single response should contain a property "ocs:share-permissions" with value "share,update,read"
```
can we also use this type of text here rather than some number? makes more readable imo

Reply:
That would be readable, but we're checking the value `share-permission` and the API responds 
``` <d:propstat>
            <d:prop>
                <share-permissions xmlns="http://open-collaboration-services.org/ns">17</share-permissions>
            </d:prop>
            <d:status>HTTP/1.1 200 OK</d:status>
        </d:propstat>
```